flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -231,6 +231,27 @@ public class YarnConfigOptions {
                     .withDescription(
                             "A comma-separated list of tags to apply to the Flink YARN application.");
 
+    /**
+     * Users and groups to give VIEW access.
+     * https://www.cloudera.com/documentation/enterprise/latest/topics/cm_mc_yarn_acl.html
+     */
+    public static final ConfigOption<String> APPLICATION_VIEW_ACLS =
+            key("yarn.view.acls")
+                    .stringType()
+                    .defaultValue("")
+                    .withDescription(
+                            "Users and groups to give VIEW access. The ACLs are of for"
+                                    + " comma-separated-usersspacecomma-separated-groups");
```

Comment:
We should modify the description here to:
```
Users and groups to give VIEW access. The ACLs are of for comma-separated-users&lt;space&gt;comma-separated-groups. Wildcard ACL is also supported. The only valid wildcard ACL is *, which grants permission to all the groups and users. 
```
The HTML files are generated from the description here.

Also, probably should mention the wildcard as well.

Reply:
Fixed.